app/views/About.js

```diff
@@ -23,18 +25,60 @@ class AboutScreen extends Component {
   }
 
   handleBackPress = () => {
+    this.setState({ tapCount: 0 });
     this.backToMain();
     return true;
   };
 
   componentDidMount() {
+    GetStoreData(DEBUG_MODE).then(dbgMode => {
+      if (dbgMode == 'true') {
+        this.setState({ tapCount: 4 });
+      }
+    });
+
     BackHandler.addEventListener('hardwareBackPress', this.handleBackPress);
   }
 
   componentWillUnmount() {
     BackHandler.removeEventListener('hardwareBackPress', this.handleBackPress);
   }
 
+  handleTapTeam = () => {
+    this.setState({ tapCount: this.state.tapCount + 1 });
+    if (this.state.tapCount >= 3) {
+      if (this.state.tapCount == 3) {
+        // Debug mode on
+        SetStoreData(DEBUG_MODE, 'true');
```

Comment:
pop up a toast notification (if we have that?) saying demo mode has been enabled.

Reply:
Don't know about this.  Sounds pretty fancy for a debugging thingy...  :)